### ISSUE TITLE
[TS] Support arg params

### ISSUE THREAD

**Issue by PalumboN** [uqbar-project/wollok#1332]
v1.6.4
![param_args](https://user-images.githubusercontent.com/4098184/36807098-c3022ce4-1ca0-11e8-90e8-031d1f680ea3.gif)
(No errors appears)

**Comment by fdodino** [comment]
Variable arguments, @PalumboN ?

**Comment by PalumboN** [comment]
Yes! @fdodino. I always forget the name (`params` is the C# keyword :P)

**Comment by fdodino** [comment]
piol! :smiley:
